uninstall.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: There must be no blank lines before the file comment (*Squiz.Commenting.FileComment.SpacingAfterOpen*).